packages/www/lib/client.tsx

```diff
@@ -9,6 +9,9 @@ const options = {
   // token: process.env.SANITY_API_TOKEN,
 };
 
+if (process.env.NODE_ENV === "development")
+  options["token"] = process.env.SANITY_API_TOKEN;
```

Comment:
Let's please wrap it inside curly brackets. Just to keep it consistent with the rest of the codebase.

Reply:
updated this on the latest commit